{{cookiecutter.github_repository}}/provisioner/roles/postgresql/tasks/main.yml

```diff
@@ -29,11 +34,6 @@
     template: 'template0'
```

Comment:
`template0` is being used to setup database and in your new command it is `template1` ?